tinystacks/ops-core
# Cleanup package

## Pull Request Type

## Summary of Patch
1. Update ops-model to latest
2. Remove unused packages left over from when API was part of this repo
3.  Check for version change type file in QA
